src/EbmlElement.cpp

```diff
@@ -245,18 +245,7 @@ EbmlElement::EbmlElement(uint64 aDefaultSize, bool bValueSet)
   Size = DefaultSize;
 }
 
-EbmlElement::EbmlElement(const EbmlElement & ElementToClone)
-  :Size(ElementToClone.Size)
-  ,DefaultSize(ElementToClone.DefaultSize)
-  ,SizeLength(ElementToClone.SizeLength)
-  ,bSizeIsFinite(ElementToClone.bSizeIsFinite)
-  ,ElementPosition(ElementToClone.ElementPosition)
-  ,SizePosition(ElementToClone.SizePosition)
-  ,bValueIsSet(ElementToClone.bValueIsSet)
-  ,DefaultIsSet(ElementToClone.DefaultIsSet)
-  ,bLocked(ElementToClone.bLocked)
-{
-}
+EbmlElement::EbmlElement(const EbmlElement & ElementToClone) = default;
```

Comment:
While I agree with the change, in "[clang-tidy] default destructor" you moved the `= default` to the class declaration. Why not do the same here with these changes here?

Reply:
I'll check again if it causes a problem.

Reply:
I think I did as you asked.

Reply:
Take a look at what you did in [this commit](https://github.com/Matroska-Org/libebml/pull/67/commits/7ef1e621d0e94aa89bc70c0c0a08de00f5be398a). Here you moved the `…::… = default` from the `.cpp` to the `.h`. That's what I meant. You could do that in this commit here as well.

Reply:
Yeah I did it in the default destructor commit. Should I move the changes to use default?